curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b4a1adb5a4304654d0347.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 646b4a1adb5a4304654d0347
+title: Step 16
+challengeType: 0
+dashedName: step-16
+---
+
+# --description--
+
+Create a `select` HTML element with the `name` attribute set to `date-options` and the `id` attribute set to `date-options`.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed